$NetBSD: patch-tools_boost__install_boost-install.jam,v 1.1 2026/02/08 22:36:16 wiz Exp $

path.is-rooted $(cmakedir) is always true in pkgsrc but generates references to
the working directory. This is useless anyway so it seems fine to drop this.

--- tools/boost_install/boost-install.jam.orig	2025-12-03 13:46:48.000000000 +0000
+++ tools/boost_install/boost-install.jam
@@ -797,7 +797,7 @@ rule make-cmake-config ( target : sources * : properti
         "get_filename_component(_BOOST_CMAKEDIR \"${CMAKE_CURRENT_LIST_DIR}/../\" REALPATH)"
         : true ;
 
-    if [ path.is-rooted $(cmakedir) ]
+    if ""
     {
         local cmakedir-native = [ path-native-fwd $(cmakedir) ] ;
 
